backend/app/api/src/helpers/UitpasTokenRepository.ts

```diff
@@ -0,0 +1,102 @@
+                        message: 'UiTPAS API is not configured',
+                        human: $t('UiTPAS is niet volledig geconfigureerd. Neem contact op met de platformbeheerder.'),
+                    });
+                }
+                uitpasToken = new UitpasToken();
+                uitpasToken.clientId = STAMHOOFD.UITPAS_API_CLIENT_ID;
+                uitpasToken.clientSecret = STAMHOOFD.UITPAS_API_CLIENT_SECRET;
+                uitpasToken.organizationId = null; // null means platform
+            }
+            else {
+                throw new SimpleError({
+                    code: 'uitpas_token_not_found',
+                    message: organizationId ? `Uitpas token not found for clientId ${organizationId}` : 'Uitpas token not found for platform',
+                    human: $t(`De UiTPAS integratie is niet compleet, contacteer de beheerder.`),
+                });
+            }
+        }
+        const newRepo = new UitpasTokenRepository(uitpasToken);
+        this.knownTokens.set(organizationId, newRepo);
+        return newRepo.getAccessToken(true);
+    }
+
+    private async getAccessToken(forceRefresh: boolean = false): Promise<string> {
+        if (forceRefresh || !this.accessToken || this.expiresOn < new Date()) {
+            const url = 'https://account-test.uitid.be/realms/uitid/protocol/openid-connect/token';
+            const myHeaders = new Headers();
+            myHeaders.append('Content-Type', 'application/x-www-form-urlencoded');
+            const params = new URLSearchParams({
+                grant_type: 'client_credentials',
+                client_id: this.uitpasToken.clientId,
+                client_secret: this.uitpasToken.clientSecret,
+            });
+            const requestOptions: RequestInit = {
+                method: 'POST',
+                headers: myHeaders,
+                body: params.toString(),
+            };
+            const response = await fetch(url, requestOptions).catch((error) => {
+                // Handle network errors
+                throw new SimpleError({
+                    code: 'uitpas_unreachable',
+                    message: `Network error when fetching UiTPAS token for client id ${this.uitpasToken.clientId}: ${error.message}`,
+                    human: $t(`We konden UiTPAS niet bereiken. Probeer het later opnieuw.`),
+                });
+            });
+            if (!response.ok) {
+                // Handle non-200 responses
+                throw new SimpleError({
+                    code: 'unsuccessful_response_fetching_uitpas_token',
+                    message: `Could not successfully obtain an UiTPAS token for client id ${this.uitpasToken.clientId}: ${response.statusText}`,
+                    human: $t(`Er is een fout opgetreden bij het verbinden met UiTPAS. Probeer het later opnieuw.`),
+                });
+            }
+            const json = await response.json().catch((error) => {
```

Comment:
```suggestion
            const json: unknown = await response.json().catch((error) => {
```
Zal ervoor zorgen dat je ook access_token moet checken op 'string'